.drone.yml

```diff
@@ -1,559 +1,325 @@
+    - ctest $CTEST_FLAGS
 ---
 kind: pipeline
-name: arm64_clang_cmake_sanitize
-
-platform:
-  os: linux
-  arch: arm64
-
+name: arm64-dynamic-clang6
+platform: { os: linux, arch: arm64 }
 steps:
 - name: Build and Test
   image: ubuntu:18.04
   environment:
-    CC: clang
-    CXX: clang++
-    CMAKE_FLAGS: -DSIMDJSON_SANITIZE=ON
+    CC: clang-6.0
+    CXX: clang++-6.0
+    CMAKE_FLAGS: -DSIMDJSON_BUILD_STATIC=OFF
+    BUILD_FLAGS: -- -j
+    CTEST_FLAGS: -j4 --output-on-failure
   commands:
     - apt-get update -qq
-    - apt-get install -y clang make cmake git
-    - $CC --version
-    - mkdir build && cd build
+    - apt-get install -y clang cmake git
+    - mkdir build
+    - cd build
     - cmake $CMAKE_FLAGS ..
-    - make -j
-    - ASAN_OPTIONS="detect_leaks=0" ctest -j4 --output-on-failure -E checkperf
+    - cmake --build . $BUILD_FLAGS
+    - ctest $CTEST_FLAGS
 ---
 kind: pipeline
-name: arm64_gcc_cmake_sanitize
-
-platform:
-  os: linux
-  arch: arm64
-
+name: arm64-sanitize-gcc8
+platform: { os: linux, arch: arm64 }
 steps:
 - name: Build and Test
   image: gcc:8
   environment:
+    BUILD_FLAGS: -- -j
+    CTEST_FLAGS: -j4 --output-on-failure -E checkperf
     CC: gcc
     CXX: g++
-    CMAKE_FLAGS: -DSIMDJSON_SANITIZE=ON
   commands:
     - apt-get update -qq
-    - apt-get install -y cmake
-    - $CC --version
-    - mkdir build && cd build
+    - apt-get install -y cmake 
+    - mkdir build
+    - cd build
     - cmake $CMAKE_FLAGS ..
-    - make -j
-    - ASAN_OPTIONS="detect_leaks=0" ctest -j4 --output-on-failure -E checkperf
-
+    - cmake --build . $BUILD_FLAGS
+    - ASAN_OPTIONS="detect_leaks=0" ctest $CTEST_FLAGS -L acceptance -LE per_implementation
+    - SIMDJSON_FORCE_IMPLEMENTATION=arm64 ASAN_OPTIONS="detect_leaks=0" ctest $CTEST_FLAGS -L per_implementation
+    - SIMDJSON_FORCE_IMPLEMENTATION=fallback ASAN_OPTIONS="detect_leaks=0" ctest $CTEST_FLAGS -L per_implementation
+    - ASAN_OPTIONS="detect_leaks=0" ctest $CTEST_FLAGS -LE "acceptance|per_implementation" # Everything we haven't run yet, run now.
 ---
 kind: pipeline
-name: amd64_clang_cmake_sanitize
-
-platform:
-  os: linux
-  arch: amd64
-
+name: arm64-sanitize-clang6
+platform: { os: linux, arch: arm64 }
 steps:
 - name: Build and Test
   image: ubuntu:18.04
   environment:
-    CC: clang
-    CXX: clang++
+    CC: clang-6.0
+    CXX: clang++-6.0
     CMAKE_FLAGS: -DSIMDJSON_SANITIZE=ON
+    BUILD_FLAGS: -- -j
+    CTEST_FLAGS: -j4 --output-on-failure -E checkperf
   commands:
-    - apt-get update -y
-    - apt-get install -y make $CC g++ cmake git
-    - $CC --version
-    - mkdir build && cd build
+    - apt-get update -qq
+    - apt-get install -y clang cmake git
+    - mkdir build
+    - cd build
     - cmake $CMAKE_FLAGS ..
-    - make -j
-    - ASAN_OPTIONS="detect_leaks=0" ctest -j4 --output-on-failure -E checkperf
+    - cmake --build . $BUILD_FLAGS
+    - ASAN_OPTIONS="detect_leaks=0" ctest $CTEST_FLAGS -L acceptance -LE per_implementation
```

Comment:
what is this detect_leaks=0 about?

Reply:
I really don't know--it was on the command lines for sanitize before too :)